src/Symfony/Component/HttpKernel/EventListener/DebugHandlersListener.php

```diff
@@ -138,19 +137,7 @@ public function configure(Event $event = null)
         }
         if ($this->exceptionHandler) {
             if ($handler instanceof ErrorHandler) {
-                $h = $handler->setExceptionHandler('var_dump');
-                if (\is_array($h) && $h[0] instanceof ExceptionHandler) {
-                    $handler->setExceptionHandler($h);
-                    $handler = $h[0];
-                } else {
-                    $handler->setExceptionHandler($this->exceptionHandler);
-                }
-            }
-            if ($handler instanceof ExceptionHandler) {
-                $handler->setHandler($this->exceptionHandler);
-                if (null !== $this->fileLinkFormat) {
-                    $handler->setFileLinkFormat($this->fileLinkFormat);
-                }
+                $handler->setExceptionHandler($this->exceptionHandler);
```

Comment:
shouldn't be `$this->exceptionHandler` be used instead of `$this->errorRenderer` below, and decouple the rendering using proper DI?